### ISSUE TITLE
some signs that need to be developed

### ISSUE THREAD

**Issue by A3ons** [bjfletcher/tech-signs#8]
HI, I am researching signs being used in ou STEM (Science, Technology, Enginering andMathematics) curriculum. 
Love your page!! 
Some suggested signs to research...  
rendering. - <-- I am researching this now 
Technology Glossary:http://v7-5.australiancurriculum.edu.au/technologies/digital-technologies/glossary#
--
This is what I am working on.. currently at Rendering :) 

Antony ..

**Comment by bjfletcher** [comment]
Hello, @A3ons! How fantastic is that! Good luck developing all the signs! By all means, my signs here aren't official – there will be dialect signs too, not forgetting that BSL & Auslan are two big but somewhat different dialects.

**Comment by A3ons** [comment]
I agree, but if the signs are clear (ie drafting - drawling line with a square-  we could use as well.
It also gives me ideas on how to sign it in Auslan.. Thanks for your hard work :)


Antony Daamen
Sum-it-up Bookkeeping
www.sum-it-up.com.au
M: 04 03 02 7654
E: a3ons@live.com.au
TRR/TRF: treasurer@townsvilleroadrunners.com.au

________________________________

**Comment by A3ons** [comment]
HI, did you comeup with something for rendering?
what about a vector pic vs a bitmap or JPG pic. 
**how to sign 'vector' ?**
bitmap was sign for
